### PR TITLE
avoid error if xclip is not reachable

### PR DESCRIPTION
**Pull request by jee-r**

### CODE REVIEW

File: shareLinkCreator [hunk 1]
```diff
@@ -232,10 +234,23 @@ if uploadFiles $url "$@"; then
...
-output="File uploaded successfully. Following public link was generated and copied to your clipboard: $shareLink"
+if command -v xclip >/dev/null 2>&1; then
+    output="File uploaded successfully. Following public link was generated and copied to your clipboard: $shareLink"
+else
+    output="File uploaded successfully. Following public link was generated: $shareLink"
+fi
+
 if [ $usenotify -eq 1 ]; then
     # Lazy hack for xfce4-notifyd cutting off long text
+<<<<<<< HEAD
```
Comment (schiessle) [inline shareLinkCreator:245]: I'm sure this doesn't belong here ;)

File: shareLinkCreator [hunk 2]
```diff
@@ -232,10 +234,23 @@ if uploadFiles $url "$@"; then
...
     output="File uploaded successfully. Following public link was generated\nand copied to your clipboard:\n$shareLink"
+=======
```
Comment (schiessle) [inline shareLinkCreator:247]: same here...

File: shareLinkCreator [hunk 3]
```diff
@@ -232,10 +234,23 @@ if uploadFiles $url "$@"; then
...
+    if command -v xclip >/dev/null 2>&1; then
+        output="File uploaded successfully. Following public link was generated\nand copied to your clipboard:\n<a href='$shareLink'>$shareLink</a>"
+    else
+        output="File uploaded successfully. Following public link was generated:\n<a href='$shareLink'>$shareLink</a>"
+    fi
+>>>>>>> e63cb31... avoid error if xclip is not reachable
```
Comment (schiessle) [inline shareLinkCreator:253]: and here....

**Comment by schiessle** [comment]
Thanks for your contribution. Can you fix the problems mentioned above and rebase it on the latest master? Thanks a lot!

**Review by schiessle — CHANGES_REQUESTED** [review]

**Comment by jee-r** [comment]
oops sorry i don't double check this one, thank you for reviewing my PR.
Maybe i should close this one and create a new it would be easier for me, i'm not really comfortable with rebase ..etc. 
i'm learning :)

**Comment by schiessle** [comment]
>Maybe i should close this one and create a new it would be easier for me, i'm not really comfortable with rebase ..etc.
i'm learning :)

sure, that's also possible

**Pull request closed without merging**